convert2rhel/unit_tests/conftest.py

```diff
@@ -117,8 +140,11 @@ def system_cert_with_target_path(monkeypatch, tmpdir, request):
     .. seealso::
         https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#using-markers-to-pass-data-to-fixtures
     """
+    if pytest.__version__.split(".") <= ["3", "6", "0"]:
+        mark = request.node.get_marker("cert_filename")
+    else:
+        mark = request.node.get_closest_marker("cert_filename")
```

Comment:
We have a utility function to replace this piece of code later, but we depend on the merge of #540.

https://github.com/oamg/convert2rhel/commit/1d003704b4bb877b800341531be0d0a462fc7a29#diff-2fe0d215565b8bb5e077a6a33b901f16a81ec51a76d5ddbb3517c2f98abaedf2R112